doc/quickstart/install.rst

```diff
@@ -22,7 +22,7 @@ and install ESMValCore into it with a single command:
 
 .. code-block:: bash
 
-    conda create --name esmvalcore -c conda-forge esmvalcore 'python=3.10'
+    conda create --name esmvalcore -c conda-forge esmvalcore 'python=3.12'
```

Comment:
How about removing the python installation altogether? It will be pulled in as a dependency anyway.

```suggestion
    conda create --name esmvalcore -c conda-forge esmvalcore
```

Reply:
totally agreed, bud!